sides/jollof-rice.md

```diff
@@ -26,12 +26,12 @@ A Ghanaian version of this spicy staple, eat it with some sweet plantains and a
 
 2. Add 2 tablespoons of oil to a blender and a chunk of onion and pulse until smooth. Then add another onion chunk and repeat until the onions look like apple sauce. Open a window, as this may sting your eyes. Transfer onion slurry to a medium bowl.
 
-2. Add the 2 cans diced tomatoes, 1 can tomato paste, and habanero pepper to the blender. Pulse until smooth. Transfer to a separate medium bowl.
-
 3. Heat the remaining 1/3 cup (80 ml) of oil in a large, heavy-bottomed pot over medium heat.
 
 4. Once the oil is shimmering, add the onion puree and cook until the water has cooked out and the puree starts to brown, about 15 minutes.
 
-5. Stir in the tomato puree and add the 2 teaspoons curry powder, 1 teaspoon garlic powder, 1 teaspoon ginger, 1/2 teaspoon dried herbs, and 3 crushed bouillon cubes. Cook for 20-30 minutes, stirring occasionally, until the stew has reduced by half and is deep red in color.
+5. While the puréed onion is sautéing, add the 2 cans diced tomatoes, 1 can tomato paste, and habanero pepper to the blender. Pulse until smooth. Transfer to a separate medium bowl.
```

Comment:
Just minor detail: should you put the mix in a bowl if the next step is to add it to the pot?